front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -102,6 +102,8 @@ const ManchetteWithSpaceTimeChartWrapper = ({
 
   const [tmpSelectedTrain, setTmpSelectedTrain] = useState(selectedTrainScheduleId);
 
+  const [previousPanning, setPreviousPanning] = useState(false);
```

Comment:
Could we use a `useRef` instead to avoid rerender ?
```suggestion
  const previousPanningRef = useRef(false);
```